docs/types/value-types.rst

```diff
@@ -224,8 +224,8 @@ Operators:
     To reduce conversion ambiguity, starting with version 0.4.24, the compiler will force you to make the truncation explicit in the conversion.
     Take for example the 32-byte value ``0x111122223333444455556666777788889999AAAABBBBCCCCDDDDEEEEFFFFCCCC``.
 
-    You can use ``address(uint160(bytes20(b)))``, which results in ``0x111122223333444455556666777788889999aAaa``,
-    or you can use ``address(uint160(uint256(b)))``, which results in ``0x777788889999AaAAbBbbCcccddDdeeeEfFFfCcCc``.
+    You can use ``address(uint160(bytes20(b)))``, which results in ``0x111122223333444455556666777788889999AAAA``,
+    or you can use ``address(uint160(uint256(b)))``, which results in ``0x777788889999AAAABBBCCCCCDDDDEEEEFFFFCCCC``.
```

Comment:
This one's not the same as the original version.

Reply:
Take for example the 32-byte value ``0x111122223333444455556666777788889999AAAABBBBCCCCDDDDEEEEFFFFCCCC``.
Different from this. The letters are in upper case.